grant-project/zcash-grant-system
# Markdown driven static pages

Closes #14.

Feel free to push style tweaks to `components/MarkdownPage/index.less`.

### What this does
- adds `html-loader` & `markdown-loader` for webpack `.md` file conversion to HTML
- adds `MarkdownPage` component to render the pages
- converts about, contact, terms of service & privacy policy to .md
- all markdown pages live @ `frontend/client/static/markdown`

### Testing
- `yarn` to update loader deps
- restart `yarn dev` 
- check that about, contact, terms of service & privacy policy pages render properly
